bench-tps/src/bench.rs

```diff
@@ -136,6 +141,7 @@ where
         use_randomized_compute_unit_price: bool,
         instruction_padding_config: Option<InstructionPaddingConfig>,
         num_conflict_groups: Option<usize>,
+        lookup_table_address: Option<(Pubkey, Pubkey)>,
```

Comment:
unclear here what the 2 pubkeys in the pair are. wdyt of making wrapper struct with `lookup_table_address` and `sbf_program_id` fields?